tests/test_prettytable.py

```diff
@@ -1912,6 +1912,106 @@ def test_jupyter_repr(self, row_prettytable: PrettyTable):
         assert row_prettytable._repr_html_() == row_prettytable.get_html_string()
 
 
+class TestMinTableWidth:
+    def test_even_min_table_width_one_column(self):
+        for i in range(15):
+            x = prettytable.PrettyTable()
+            x.field_names = ["Test table"]
+            x.add_row(["X" * i])
```

Comment:
Please could you use a more descriptive variable name for this? For example:
```suggestion
        for col_width in range(15):
            x = prettytable.PrettyTable()
            x.field_names = ["Test table"]
            x.add_row(["X" * col_width])
```

And below.

Reply:
I used your suggestion